messages/german.json

```
@@ -0,0 +1,1387 @@
+{
+    "_metadata": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Metadata",
+        "fallback": "en",
+        "articles": [
+            {"pattern": "[aeiou]*", "article": "einen"},
+            {"pattern": null, "article": "ein"}
+        ],
+        "plural": [
+            {"number": 1, "index": 0},
+            {"number": null, "index": 1}
+        ],
+        "list": [
+            " und ",
+            ", ",
+            ", und "
+        ]
+    },
+    "_roles": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Roles",
+        "alpha wolf": ["Leitwolf", "Leitwölfe"],
+        "amnesiac": ["Vergessener", "Vergessener"],
+        "assassin": ["Assassine", "Assassinen"],
+        "augur": ["Weisssager", "Weisssager"],
+        "blessed villager": ["Gesegneter", "Gesegnete"],
```

Bitte das "Dorfbewohner" hier und anderswo nicht aussparen.

```suggestion
        "blessed villager": ["Gesegneter Dorfbewohner", "Gesegnete Dorfbewohner"],
```

---

Dorfbewohner wurde in Bürger umgeändert diese änderung sollte bereits sichtbar sein in der german.json